src/main/scala/net/snowflake/spark/snowflake/pushdowns/querygeneration/MiscStatement.scala

```diff
@@ -135,6 +136,14 @@ private[querygeneration] object MiscStatement {
           case None => EmptySnowflakeSQLStatement()
         }} + ConstantString("END")
 
```

Comment:
To reviewers:
1. The original fix is done in https://github.com/snowflakedb/spark-snowflake/pull/253 .
2. The original fix is reverted before releasing SC 2.8.2 in https://github.com/snowflakedb/spark-snowflake/pull/295 .
3. Use “git cherry-pick <hash_code>” to bring back the fix. No merge conflicts happens. Test is PASS.